workflow/snakemake_rules/nextclade.smk

```diff
@@ -193,9 +193,15 @@ rule nextclade_info:
         nextclade_info = f"data/{database}/nextclade{{reference}}.tsv"
     shell:
         """
-        # Header taken from first non-empty file
-        keep-header {input.old_info:q} {input.new_info:q} -- \
-        tsv-uniq -f 1 > {output.nextclade_info}
+        TMPFILE="data/{database}/nextclade{{reference}}.tmp.tsv"
+        if [[ -s {input.old_info} ]]; then
+            mv {input.old_info} $TMPFILE
+            tail -n +2 {input.new_info} >> $TMPFILE
+        else
+            mv {input.new_info} $TMPFILE
+        fi
+        tsv-uniq -H -f seqName $TMPFILE > {output.nextclade_info}
+        rm $TMPFILE
```

Comment:
Hmm, I'm worried about disk space with the `TMPFILE`. 

Is there any reason we can't use `tsv-append`?
```
tsv-append -H {input.old_info} {input.new_info} \
    | tsv-uniq -H -f seqName > {output.nextclade_info}
```

Reply:
I had no idea `tsv-append` existed! That's neat!

Should work.

Reply:
Regarding diskspace, the tsv is only something like 10-20GB uncompressed, peanuts compared to the fasta which is maybe 400 GB but in any case this is nicer!

Reply:
I haven't tried, but is it not enough to just replace `1` to `seqName` in the original code? Why other changes are necessary?

```bash
keep-header {input.old_info:q} {input.new_info:q} -- \
        tsv-uniq -f seqName > {output.nextclade_info}
```

(notice `-f seqName` instead of `-f 1`)